.github/workflows/close-stale-prs.yml

```diff
@@ -5,7 +5,7 @@ on:
 
 jobs:
   close-issues:
-    runs-on: ubuntu-latest
+    runs-on: [self-hosted, Linux, X64]
```

Comment:
Just to be sure, in this PR you are not using large your intention is not to use `single-cell-8c64g-runner` or `single-cell-1tb-runner` runners.

Reply:
correct - this workflow can run on a much smaller runner. This set of labels is based on Jake's previous advice (see py-unittests.yml for an example)